external-builds/pytorch/build_prod_wheels.py

```diff
@@ -602,9 +602,31 @@ def do_build_pytorch(
     pytorch_build_version = (pytorch_dir / "version.txt").read_text().strip()
     pytorch_build_version += args.version_suffix
     print(f"  Default PYTORCH_BUILD_VERSION: {pytorch_build_version}")
-    print(
-        f"  Flash attention enabled: {args.enable_pytorch_flash_attention_windows or not is_windows}"
-    )
+
+    # Enable/disable only for non-Windows
+    if not is_windows:
+        if "2.7" in str(pytorch_build_version):
+            env["USE_FBGEMM_GENAI"] = "ON"
+            use_flash_attention = (
+                "1" if args.enable_pytorch_flash_attention_linux else "0"
+            )
```

Comment:
I think we may want to invert this condition. Only disable on 2.10+ (for now). Keep it enabled on all other versions by default. We should be able to use this script to build 2.8 and 2.9 also.